underscore.js

```diff
@@ -1446,6 +1453,13 @@
     return _.isFunction(value) ? value.call(object) : value;
   };
 
+  // Returns the, possibly deeply nested, value at `path` if defined.
+  // Otherwise, returns `fallback`.
+  _.get = function(object, path, fallback) {
+    var value = property(path)(object);
+    return value === void 0 ? fallback : value;
+  };
```

Comment:
Okay, I'm aware you mentioned them, I didn't phrase my response well so apologies for that (was rushing to submit a paper). What I meant is what is the motivation and value of adding `_.get` (and committing to `_.set` and `_.unset`) rather than modifying `_.result` for paths.